metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/usage.py

```diff
@@ -845,7 +860,7 @@ def _get_parsed_bigquery_log_events(
     ) -> Iterable[AuditEvent]:
         parse_fn: Callable[[Any], Optional[AuditEvent]]
         if self.config.use_exported_bigquery_audit_metadata:
-            bq_client = BigQueryClient(project=project_id)
+            bq_client = BigQueryClient()
```

Comment:
I believe , we should use  `datahub.ingestion.source.bigquery_v2.common.get_bigquery_client` here.

Reply:
We didn't used to do this but this seems reasonable, adding